internal/backend/local/local.go

```diff
@@ -92,8 +92,8 @@ func (b *Local) Save(ctx context.Context, h restic.Handle, rd restic.RewindReade
 
 	defer func() {
 		// Mark non-retriable errors as such (currently only
-		// "no space left on device").
-		if errors.Is(err, syscall.ENOSPC) {
+		// "no space left on device" and "Permission denied").
```

Comment:
Maybe the parenthesized part of the comment can go now. There's no use enumerating the cases in both the code and the comment.